### PR TITLE
feat(storybook): render hex colors in stories LS-1868

### PR DESCRIPTION
**Pull request by remarkablemark**
## Jira

[Render hex colors in design-system Storybook](https://littlespoon.atlassian.net/browse/LS-1868)

## Motivation

feat(storybook): render hex colors in stories

## Current Behavior

- Only rgba colors displayed

## New Behavior

- Hex colors displayed in stories
- Grouped stories into themes
- Make theme fonts readonly
- Deleted unused assets in storybook

## Test Plan

https://deploy-preview-87--littlespoon.netlify.app/?path=/story/design-system-theme-colors--token

## Checklist

- [x] [Conventional Commits](https://www.conventionalcommits.org/)
- [x] Storybook

### CODE REVIEW

File: packages/storybook/package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "build": "yarn copy-changelog && build-storybook",
     "clean": "rm -rf storybook-static src/stories/changelog",
     "copy-changelog": "mkdir -p src/stories/changelog && ts-node scripts/copy-changelog",
-    "lint": "eslint .",
-    "lint:fix": "yarn lint --fix",
+    "lint": "eslint . && tsc --noEmit",
```
Comment (remarkablemark) [inline packages/storybook/package.json:10]: I realized Storybook wasn't throwing TypeScript compilation errors during build time so I added it here

File: packages/storybook/src/stories/Divider.stories.tsx [hunk 2]
```diff
@@ -13,13 +13,11 @@ const Template: ComponentStory<typeof Divider> = (args) => (
...
-DividerOne.args = {
-  inverted: false,
-}
+export const DividerDefault = Template.bind({})
+DividerDefault.args = {}
 
-export const DividerTwo = Template.bind({})
-DividerTwo.args = {
+export const DividerInverted = Template.bind({})
+DividerInverted.args = {
```
Comment (remarkablemark) [inline packages/storybook/src/stories/Divider.stories.tsx:20]: Simplified the Divider stories

File: packages/storybook/src/stories/theme/Colors.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                      style={{
+                        backgroundColor: color,
+                        height: 100,
+                        width: 100,
+                      }}
+                    />
+
+                    <td style={{ padding: '1rem' }}>
+                      <code>
+                        {hex.toUpperCase()}
```
Comment (remarkablemark) [inline packages/storybook/src/stories/theme/Colors.tsx:41]: Added color hex

File: packages/theme/src/fonts/index.ts [hunk 4]
```diff
@@ -8,28 +8,30 @@ const rems: Record<string, string> = {}
 
 const rem = (value: number): string => rems[value] || (rems[value] = `${value}rem`)
 
+const fontFamily = (name: string): string => `${name}, sans-serif`
+
 const xLargeParagraph = {
   fontSize: rem(2),
   lineHeight: rem(3),
-}
+} as const
```
Comment (remarkablemark) [inline packages/theme/src/fonts/index.ts:16]: Make these readonly const

File: packages/ui/package.json [hunk 5]
```diff
@@ -7,7 +7,7 @@
     "build": "tsc",
     "build:flowtypes": "find . -type f -not -path './node_modules/*' -name '*.d.ts' -exec bash -c 'yarn flowgen --add-flow-header $1 -o ${1%.*.*}.js.flow' _ '{}' \\;",
     "clean": "rm -rf coverage && find . \\( -name '*.js' -o -name '*.js.map' -o -name '*.js.flow' -o -name '*.d.ts*' \\) -type f -delete",
-    "lint": "eslint .",
+    "lint": "eslint --ignore-path .gitignore .",
```
Comment (remarkablemark) [inline packages/ui/package.json:10]: The ignore-path was not set

File: packages/storybook/src/stories/theme/Fonts.tsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+              <Divider />
+            </Fragment>
+          ),
+        )
+      })}
+    </main>
+  )
+}
+
+function getFontName(font: string): string {
```
Comment (hewbahrami) [inline packages/storybook/src/stories/theme/Fonts.tsx:75]: Are we using any documentation for functions in DS?

Comment (remarkablemark) [reply packages/storybook/src/stories/theme/Fonts.tsx:75]: We don't but we can add a rule

Comment (hewbahrami) [reply packages/storybook/src/stories/theme/Fonts.tsx:75]: I think it would be nice. I like the JS documentation we've adopted in the other repos.

Comment (remarkablemark) [reply packages/storybook/src/stories/theme/Fonts.tsx:75]: For this one, you want me to do?

```ts
/**
 * Get font name
 */
function getFontName(font: string): string {
```

**Review by hewbahrami — APPROVED** [review]

File: packages/storybook/src/stories/theme/Fonts.tsx (hunk 6 above)
Comment (remarkablemark) [reply packages/storybook/src/stories/theme/Fonts.tsx:75]: Added TSDoc c81f828

**Pull request merged**